crates/sui-bridge-indexer/src/models.rs

```diff
@@ -11,6 +11,13 @@ pub struct ProgressStore {
     pub checkpoint: i64,
 }
 
+#[derive(Queryable, Selectable, Insertable, Identifiable, Debug)]
+#[diesel(table_name = sui_progress_store, primary_key(txn_digest))]
+pub struct SuiProgressStore {
```

Comment:
it's for tracking query cursor as you know. The cursor is an TransactionDigest so better to be stored as bytes. We can tweak the existing table `progress_store` to consolidate, but i don't know if we want to do it now (cuz it may break currently running indexer if we are not careful enough)